client/my-sites/google-my-business/google-my-business-stats/index.js

```diff
@@ -0,0 +1,320 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import page from 'page';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from 'components/section-header';
+import Button from 'components/button';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import Main from 'components/main';
+import Card from 'components/card';
+import StatsNavigation from 'blocks/stats-navigation';
+import { recordTracksEvent } from 'state/analytics/actions';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormSelect from 'components/forms/form-select';
+
+class Stats extends Component {
+	static propTypes = {
+		recordTracksEvent: PropTypes.func.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	goBack = () => {
+		page.back( `/google-my-business/${ this.props.siteId }` );
+	};
+
+	render() {
+		const { translate } = this.props;
+		return (
+			<Main>
+				<SidebarNavigation title="Stats" />
+				<div className="google-my-business-stats">
+					<StatsNavigation selectedItem={ 'google-my-business' } />
+
+					<Card>
+						<div className="google-my-business-stats__container">
+							<img
+								src="https://lh5.googleusercontent.com/p/AF1QipPBVVXFSwBfsObM5TbyoLSGySD_CJAXiztaxf0a=w408-h544-k-no"
+								className="google-my-business-stats__image"
+								alt="gravatar"
+							/>
+
+							<div className="google-my-business-stats__text">
+								<div className="google-my-business-stats__header">Cate's Cookies</div>
+								<div className="google-my-business-stats__verified">
+									<Gridicon icon="checkmark-circle" size={ 18 } /> Verified
+								</div>
+							</div>
+						</div>
+
+						<div className="google-my-business-stats__listing-detail">
+							<Gridicon icon="phone" size={ 18 } /> (865) 367-2746
+						</div>
+						<div className="google-my-business-stats__listing-detail">
+							<Gridicon icon="time" size={ 18 } /> 9:00 AM - 6:00 PM
+						</div>
+						<div className="google-my-business-stats__listing-detail">
+							<Gridicon icon="globe" size={ 18 } /> www.catescookies.com
+						</div>
+						<div className="google-my-business-stats__address">
+							<Gridicon icon="location" size={ 18 } />
+							<div className="google-my-business-stats__listing-detail">
+								345 North Avenue<br />Talihassee, FL 34342<br />USA
+							</div>
+						</div>
+						<div className="google-my-business-stats__listing-detail">
+							<Gridicon icon="tag" size={ 18 } /> Bakery
+						</div>
+						<Button
+							className="google-my-business-stats__listing-button"
+							href="https://business.google.com"
+							target="_blank"
+						>
+							Update Listing
+							<Gridicon icon="external" />
+						</Button>
+					</Card>
+
+					<SectionHeader label={ translate( 'Searches' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **58** times:
`translate( 'Top Searches' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).